webview-ui/src/components/history/HistoryPreview.tsx

```diff
@@ -33,7 +33,7 @@ const HistoryPreview = ({ showHistoryView }: HistoryPreviewProps) => {
 				)}
 				{tasks.length === 0 && (
 					<>
-						<p className="outline rounded p-4">
+						<p className=" border-[#555] p-2 text-lg text-center mx-8">
```

Comment:
not sure about using static colour here, wouldn't it be better to pick one from the theme token?

though i'm guessing it'll be just fine since it's grey/neutral colour